### ISSUE TITLE
Country IRL no longer working in 2.1.6beta6 (worked up to 2.1.5)

### ISSUE THREAD

**Issue by Gavin-A92** [RogerSelwyn/Home_Assistant_SkyQ_MediaPlayer#1]
Updated Custom Component no longer working as was in 2.1.5.
Looking at logs (below) I believe this is down to "new" feature around removing the need for Country parameter. IRL (Ireland) as a country has worked 100% perfectly up to now but no longer works in this beta. Understand this is (and never was) technically "supported" but have found it to work pretty much perfectly up to this version.
"Buttons" (change channel, etc) work as expected but the boxes (1x Q 2TB and 2 x Mini) are not correctly detected by the component (and report as off despite being on (and controllable too))

**To Reproduce**
No steps to reproduce per se, once upgraded this was noticed.
Notwithstanding support for GBR/ITA this has worked fine for IRL up to 2.1.6
Media Player(s) are detected but functionality is reduced.

**config.yaml extract**

```
media_player:
  - platform: skyq
    host: 192.168.10.241
    name: SkyQ Sitting Room
    room: Sitting Room
    config_directory: '/config/'
    generate_switches_for_channels: true
    sources:
      DisneyJnr: '6,0,7'                                                        
      DisneyJnr+1: '6,0,8'                                                      
      CHup: 'channelup'
      CHdn: 'channeldown'
      Select: 'select'                                                          
      Up: 'up'                                                                  
      Down: 'down'                                                              
      Left: 'left'                                                              
      Right: 'right'                                                            
      I: 'i'                                                                    
      Dismiss: 'dismiss'                                                        
      Sidebar: 'Sidebar'                                                        
      Home: 'home'                                                              
      TVGuide: 'tvguide'                                                        
      Sky: 'sky'                                                                
      Record: 'record'      
```                                                    
````
**Output of HA logs**
Paste the relavant output of the HA log here.

Think this is the relevant errors from the log........
2020-05-05 22:43:40 ERROR (SyncWorker_16) [pyskyqremote.skyq_remote] E0040 - Invalid country: 192.168.10.241 : IRL : No module named 'pyskyqremote.country.remote_ie'
2020-05-05 22:43:40 ERROR (SyncWorker_10) [pyskyqremote.skyq_remote] E0040 - Invalid country: 192.168.10.242 : IRL : No module named 'pyskyqremote.country.remote_ie'
2020-05-05 22:43:40 ERROR (SyncWorker_7) [pyskyqremote.skyq_remote] E0040 - Invalid country: 192.168.10.243 : IRL : No module named 'pyskyqremote.country.remote_ie'
2

**Comment by RogerSelwyn** [comment]
I'm interested that it worked with v2.1.5 because I don't think it should have done. That said, the only way I could see it working is if it was somehow defaulting to GBR (I'm not going to start digging through it all). 

Could you give me the output of this:

`http://192.168.10.241:9006/as/system/deviceinformation`

In the meantime if you sent country to 'GBR', I suspect it will work.

**Comment by Gavin-A92** [comment]
Here you go

`{
   "ASVersion" : "Q112.000.21.00-AS_asdev",
   "IPAddress" : "192.168.10.241",
   "bouquet" : 4104,
   "countryCode" : "IRL",
   "gateway" : true,
   "hardwareName" : "Falcon",
   "householdToken" : "REMOVED",
   "modelNumber" : "Q112.000.21.00L (53wk8j8)",
   "pipCapable" : true,
   "serialNumber" : "REMOVED",
   "streamingCapable" : true,
   "subbouquet" : 50,
   "versionNumber" : "32B057"
}`


I'd guess you're right around changing/setting country to GBR.
From what I know SKY setup for UK and Ireland are pretty much the same with Ireland being treated as a Region for programming rather than another country

**Comment by RogerSelwyn** [comment]
So I think you should get full functionality if you set to GBR straightaway if that is true. Should switch on/off/etc I think. But I'll include a mechanism to convert IRL to access the 'gb' code.

**Comment by Gavin-A92** [comment]
Yup, all seems back to normal once added "country" to my config....cheers

![image](https://user-images.githubusercontent.com/32313094/81121164-fc704700-8f25-11ea-8f74-e38e3f0e7965.png)

**Comment by RogerSelwyn** [comment]
Sorry for the break in service... I'll do the translation from IRL in the final release so you won't have to put that in. Now, how to make that generic..... more lookups. For tomorrow...

**Comment by Gavin-A92** [comment]
Nothing to be sorry about.....the component is fantastic.
And anyway that's what betas and testing is all about :) .....thanks again.

**Comment by RogerSelwyn** [comment]
The fix for this is in the pyskyqremote 0.2.13. The next release of SkyQ for HA will include that version.

https://github.com/RogerSelwyn/skyq_remote/commit/35395a6332fee097d3048b4e3ac1bca507a5a5b4

**Comment by Gavin-A92** [comment]
looks good....thanks

**Comment by RogerSelwyn** [comment]
Closed with release v2.1.6 - https://github.com/RogerSelwyn/Home_Assistant_SkyQ_MediaPlayer/releases/tag/v2.1.6
